src/pas/plugins/kimug/utils.py

```
@@ -202,64 +202,80 @@ def get_plugin(pluginid="oidc"):
+    access_token = _get_keycloak_admin_token()
     if not access_token:
         logger.error("Could not get access token from Keycloak")
         return []
-    kc_users = []
-    url = f"{keycloak_url}admin/realms/{realm}/users?max=100000"
+    keycloak_url = os.environ.get("keycloak_url")
+    url = f"{keycloak_url}admin/realms/{realm}/users"
+    if max_results is not None:
+        url = f"{url}?max={max_results}"
     headers = {"Authorization": "Bearer " + access_token}
     response = requests.get(url=url, headers=headers)
```

Ruff (0.15.20)

[error] Probable use of `requests` call without timeout

(S113)